src/CommonLib/LDAPUtils.cs

```diff
@@ -1452,26 +1472,33 @@ private LdapConnection CreateConnectionHelper(string directoryIdentifier, bool s
             return connection;
         }
 
+        /// <summary>
+        /// Checks the LDAP exception and throws an appropriate custom exception based on the error code.
+        /// </summary>
+        /// <param name="ldapException">The LDAP exception to check.</param>
         private static void CheckAndThrowException(LdapException ldapException)
```

Comment:
I don't know that I've actually done anything useful to the legibility of `CheckAndThrowException` 🤷